fstat/lib.py

```diff
@@ -26,3 +28,11 @@ def parse_end_date(end_date):
     else:
         end_date = datetime.strptime(end_date, '%Y-%m-%d')
     return end_date
+
+
+def get_branch_list(fid=None):
+    if not fid:
```

Comment:
One of the items in this list will be `None` when this code is deployed. We have rows where we did not store the branch information. Filter away the None.